include/circt/Dialect/Verif/VerifOps.td

```diff
@@ -346,6 +349,105 @@ def FormalOp : VerifOp<"formal", [
   }];
 }
 
+def SimulationOp : VerifOp<"simulation", [
```

Comment:
What's the rationale of having this in the `verif` dialect vs `sim` dialect?

Reply:
Yeah agreed, the lines are somewhat blurry here. I think it would make sense to keep the verification intent operations in the same dialect, since they are similar and might share documentation and implementation bits in the future. That's why I opted for adding a `verif.simulation` instead of a `sim.simulation`. I agree though that we should use the `sim` dialect for all other simulation-specific ops. For example, if we ever add something like a stimulus / procedural test sequence op, it would probably be something like `sim.execute_region { ... }` or similar.